src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        Color[] colorList = Color.values();
+        Random random = new Random();
```

Comment:
random has to be a class field, not a local variable. The reason is to not unnecessarily recreate it but to use reuse it